vue-app/src/router.ts

```diff
@@ -2,11 +2,13 @@ import { createRouter, createWebHistory } from "vue-router";
 import HomePage from "./pages/HomePage.vue";
 import TablePage from "./pages/TablePage.vue";
 import NotFound from "./pages/NotFound.vue";
+import RecursivePage from "./pages/RecursivePage.vue";
 
 export default createRouter({
   history: createWebHistory(),
   routes: [
     { path: "/", component: HomePage },
+    { path: "/RecursiveRendering", component: RecursivePage },
```

Comment:
page url should be kebap-case

Reply:
path changed to Recursive-Rendering

Reply:
kebap-case means that every word should be lowercase

Reply:
changed to lowercase

Reply:
Table should be fixed too